x-pack/plugins/snapshot_restore/public/application/components/policy_form/steps/step_logistics.tsx

```diff
@@ -54,6 +55,19 @@ export const PolicyStepLogistics: React.FunctionComponent<StepProps> = ({
   const { i18n, history } = useServices();
   const { docLinks } = useCore();
 
+  const disableToolTip = (component: ReactElement): ReactElement => {
+    return policy?.isManagedPolicy ? (
+      <EuiToolTip
+        content="This field is disabled because you are editing a managed policy."
```

Comment:
Can we add an i18n translation for this?

Reply:
Added i18n translation for both (managed repository and managed policy)